evergreen/run-tests.sh

```diff
@@ -97,4 +104,16 @@ fi
 for var in TMP TEMP NUGET_PACKAGES NUGET_HTTP_CACHE_PATH APPDATA; do
   export $var=z:\\data\\tmp;
 done
-powershell.exe .\\build.ps1 -target ${TARGET}
+
+if [[ "$CLIENT_PEM" != "nil" ]]; then
+  CLIENT_PEM=${CLIENT_PEM} source evergreen/convert-client-cert-to-pkcs12.sh
```

Comment:
The logic that was previously in evergreen.yml has now been moved here, where the `source` command works as expected